temporalio/converter.py

```diff
@@ -852,6 +852,12 @@ def _error_to_failure(
                 failure.application_failure_info.next_retry_delay.FromTimedelta(
                     error.next_retry_delay
                 )
+            if error.category:
+                failure.application_failure_info.category = (
+                    temporalio.api.enums.v1.ApplicationErrorCategory.ValueType(
+                        error.category or 0
```

Comment:
```suggestion
                        error.category
```

_Probably_ don't need `or 0`, but meh

Reply:
removed